unity/unity-embed-host/CoreCLRHost.cs

```diff
@@ -154,6 +154,11 @@ public static int array_length([NativeCallbackType("MonoArray*")] IntPtr array)
         [NativeCallbackType("MonoClass*")] IntPtr klass)
         => FormatterServices.GetUninitializedObject(klass.TypeFromHandleIntPtr()).ToNativeRepresentation();
 
+    [return: NativeCallbackType("MonoException*")]
+    public static IntPtr exception_from_class_msg([NativeCallbackType("MonoClass*")] IntPtr klass,
+        [NativeCallbackType("const char*")] sbyte* msg)
```

Comment:
Do we need to worry about a `null` msg parameter? Not sure if mono/il2cpp handle that.